eng/common/pipelines/templates/steps/bypass-local-dns.yml

```diff
@@ -6,11 +6,6 @@ steps:
       condition: |
         and(
           succeededOrFailed(),
-          or(
-            eq(variables['OSVmImage'], 'ubuntu-18.04'),
-            eq(variables['OSVmImage'], 'ubuntu-20.04'),
-            eq(variables['OSVmImage'], 'MMSUbuntu18.04'),
-            eq(variables['OSVmImage'], 'MMSUbuntu20.04')
-          ),
+          contains(variables['OSVmImage'], 'ubuntu'),
```

Comment:
@mikeharder based on your investigations do you think we should keep this specialized to 18.04 or should we try to generalize it for all ubuntu images?